Food-Recipe/styles.css

```diff
@@ -266,10 +266,7 @@ button:hover {
 
   width: 95%;
   max-height: 90%;
-  overflow-y: scroll;
-  scrollbar-width: thin;
```

Comment:
@andreimaier The  `scrollbar-width` and `scrollbar-color:` are used for styling the scrollbar for Firefox. By removing these two lines, the scrollbar's style is back to default when viewed on Firefox, so I'm going to re-add and commit to the main branch.